bundledata.go

```diff
@@ -141,6 +141,9 @@ type ServiceSpec struct {
 	// Storage holds the constraints for storage to assign
 	// to units of the service.
 	Storage map[string]string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
+
+	// EndpointBindings maps how endpoints are bound to spaces
+	EndpointBindings map[string]string `bson:"bindings,omitempty" json:"bindings,omitempty" yaml:"bindings,omitempty"`
```

Comment:
As discussed on IRC I'm surprised Storage ... `yaml ",omitempty"` works correctly for "storage:", but if it does, let's drop the "bindings" in all 3 cases for consistency.